### PR TITLE
Updated Officer Bios

### LINKED ISSUE #68: Update Officer Bios
Update team page to have new pictures and blurbs.

### PR DESCRIPTION
**Pull request by khxia**
Fixes #68.

### CODE REVIEW

File: src/components/PhotoPage/PhotoPage.js [hunk 1]
```diff
@@ -108,21 +100,44 @@ const officers = [
...
-		id: 'furn',
-		description: `Hi! I'm a fourth year CS major from Bangkok, Thailand 
-		(you can tell from my very long last name). I am a BIG fan of food and 
-		I like to listen to new jazz/R&B music. I will be co-directing Hack 
-		Kitchen with Raji this quarter so come say hi!!`
-	},
-	{
-		role: 'Officer',
-		name: 'Galen Wong',
+		name: 'Galen-senpai',
```
Comment (jodymlin) [inline src/components/PhotoPage/PhotoPage.js:103]: Change this back to Galen Wong

Comment (khxia) [reply src/components/PhotoPage/PhotoPage.js:103]: nooooo

File: src/components/PhotoPage/PhotoPage.js [hunk 2]
```diff
@@ -108,21 +100,44 @@ const officers = [
...
+		(sometimes) funny jokes and (always) hilarious puns. Currently living off of Disney movies in quarantine.
+		`
+	},
+	{
+		role: 'Advisor',
+		name: 'Connie Chen',
+		id: 'connie',
+		description: `Hello! My name is Connie, and I'm a third year Computer Science major and Cognitive Science minor. 
+		I like the California weather, messenger stickers, and music. Currently living off of Disney movies in quarantine.`
+	},
```
Comment (TimothyGu) [inline src/components/PhotoPage/PhotoPage.js:140]: It looks like there are quite a few different styles of wrapping long lines for `description`:

```js
`Abc
Def`
```
```js
`
Abc
Def
`
```
```js
`Abc
 def`
```
etc.

Can we try to settle on a single style?

File: src/components/PhotoPage/PhotoPage.js [hunk 3]
```diff
@@ -108,21 +100,44 @@ const officers = [
...
+		Hi there, I’m Lea. Come hang out with me and the rest of hack at our events (on zoom) to hear my 
+		(sometimes) funny jokes and (always) hilarious puns. Currently living off of Disney movies in quarantine.
+		`
+	},
+	{
+		role: 'Advisor',
+		name: 'Connie Chen',
+		id: 'connie',
+		description: `Hello! My name is Connie, and I'm a third year Computer Science major and Cognitive Science minor. 
+		I like the California weather, messenger stickers, and music. Currently living off of Disney movies in quarantine.`
```
Comment (TimothyGu) [inline src/components/PhotoPage/PhotoPage.js:139]:
```suggestion
		I like the California weather, Messenger stickers, and music. Currently living off of Disney movies in quarantine.`
```

Also the "Currently living off of Disney movies in quarantine." seems to be copy-pasted from Lea? It doesn't like Connie lol.

File: src/components/PhotoPage/PhotoPage.js [hunk 4]
```diff
@@ -108,21 +100,44 @@ const officers = [
...
+		id: 'miles',
+		description: `I’m a CS major passionate about hackathons and Powell library.
+		 Most of my experience deals with web/mobile dev. Catch me out here eating way too much food, drinking way too 
+		 much boba (@Jamie for enabling me), and vibing to chill music.`
+	},
+	{
+		role: 'Officer',
+		name: 'Alex Xia',
+		id: 'alex',
+		description: `Hi, I'm a first year CS major from Hong Kong. Aside from coding,
```
Comment (TimothyGu) [inline src/components/PhotoPage/PhotoPage.js:121]: Some people use straight quotes `'` but others use curly quotes `’`. Make things `’` consistently.

File: src/components/PhotoPage/PhotoPage.js [hunk 5]
```diff
@@ -108,21 +100,44 @@ const officers = [
...
+		description: `I’m a CS major passionate about hackathons and Powell library.
+		 Most of my experience deals with web/mobile dev. Catch me out here eating way too much food, drinking way too 
+		 much boba (@Jamie for enabling me), and vibing to chill music.`
+	},
+	{
+		role: 'Officer',
+		name: 'Alex Xia',
+		id: 'alex',
+		description: `Hi, I'm a first year CS major from Hong Kong. Aside from coding, 
+		I like to watch anime and play games - occasionally, Jody baits me into watching kdrama.
```
Comment (TimothyGu) [inline src/components/PhotoPage/PhotoPage.js:122]:
```suggestion
		I like to watch anime and play games – occasionally, Jody baits me into watching kdrama. 
```

Use a dash.

File: src/components/PhotoPage/PhotoPage.js [hunk 6]
```diff
@@ -24,79 +24,71 @@ const teamIntro = `
...
+		too loudly, binging Youtube videos, or making too many Spotify playlists.`
 	},
 	{
 		role: 'Officer',
 		name: 'Sahen Rai',
 		id: 'sahen',
-		description: `I'm a board member for ACM Hack. I attend meetings and plan 
-		cool events for Hack. In my free time, I like to make lame clarinet covers 
-		of pop songs.`
+		description: `Hi Im Sahen! I’m a second year student, and in addition to ACM I also participate in the Foundations dance Org :).
```
Comment (TimothyGu) [inline src/components/PhotoPage/PhotoPage.js:90]:
```suggestion
		description: `Hi I’m Sahen! I’m a second year student, and in addition to ACM I also participate in the Foundations dance Org :).
```

File: src/components/PhotoPage/PhotoPage.js [hunk 7]
```diff
@@ -24,79 +24,71 @@ const teamIntro = `
...
-		name: 'Raji Jadhav',
-		id: 'raji',
-		description: `Hey! I'm Raji, a 2nd year Hack officer, originally from 
-		India! Love to dance, love to sing, love to code. I believe in 
-		community learning, a multiverse, and fairies :) Current obsessions 
-		include ML, Thai food, and calligraphy. Tend to never stop talking.`
+		name: 'Asha Kar',
+		id: 'asha',
+		description: `Hello! I'm a first year CS major with a deep love for coffee and dogs. 
+		Besides coding, I enjoy knitting, reading, bullet journaling,  and watching sitcoms.`
```
Comment (TimothyGu) [inline src/components/PhotoPage/PhotoPage.js:60]:
```suggestion
		Besides coding, I enjoy knitting, reading, bullet journaling, and watching sitcoms.`
```

File: src/components/PhotoPage/PhotoPage.js [hunk 8]
```diff
@@ -24,79 +24,71 @@ const teamIntro = `
...
+		include ML, Thai food, and calligraphy. Tend to never stop talking.`
 	},
 	{
-		role: 'President',
-		name: 'Connie Chen',
-		id: 'connie',
-		description: `Hello! I'm Connie, a third year computer science major. 
-		I love the california weather, facebook messenger stickers, and music. 
-		Getting added to the Dogspotting fb group also changed my life (shout-out to my co Lea).`
+		role: 'Vice-President',
```
Comment (TimothyGu) [inline src/components/PhotoPage/PhotoPage.js:35]: In the US they spell it without the hyphen.

```suggestion
		role: 'Vice President',
```

**Comment by TimothyGu** [comment]
Could you also add this following to the pull request description:

```markdown
Fixes #68.
```

This is the standard way of specifying association with an issue (#68 in this case).

File: src/components/PhotoPage/PhotoPage.js (hunk 3 above)
Comment (jodymlin) [reply src/components/PhotoPage/PhotoPage.js:139]: This is what they both actually submitted for their bios :)

Comment (TimothyGu) [reply src/components/PhotoPage/PhotoPage.js:139]: Heh ok…

File: src/components/PhotoPage/PhotoPage.js (hunk 2 above)
Comment (khxia) [reply src/components/PhotoPage/PhotoPage.js:140]: `Abc
Def`

I've settled on this one

File: src/components/PhotoPage/PhotoPage.js (hunk 4 above)
Comment (khxia) [reply src/components/PhotoPage/PhotoPage.js:121]: the curly quote was a problem that occurred when copy and pasting the descriptions. I have made things consistent and will stick with `'` and `"`

**Comment by khxia** [comment]
Fixes #68

**Pull request merged**